napari/components/dims.py

```diff
@@ -45,22 +45,23 @@ class Dims(EventedModel):
     last_used : int
         Dimension which was last used.
     range : tuple of 3-tuple of float
-        List of tuples (min, max, step), one for each dimension. In a world
-        coordinates space. As with Python's `range` and `slice`, max is not
-        included.
-    current_step : tuple of int
-        Tuple the slider position for each dims slider, in slider coordinates.
+        List of tuples (min, max, step), one for each dimension in world
+        coordinates space.
+    span : tuple of 3-tuple of float
```

Comment:
```suggestion
    span : tuple of pairs of floats
```

or 2-tuples instead of pairs

Reply:
@brisvag the "span" attributes now need to be removed from the docstring, yes?

Reply:
and `left_margin` and `right_margin` need to be added. btw I might consider a sub object but that might have issues with evented model — `margin.left` and `margin.right`. Totally up to you if that makes sense to you.

Reply:
Either way though, I *would* rename them `margin_left` and `margin_right` so that they sit together naturally e.g. in an autocomplete.

Reply:
Should I also go to `margin_left_experimental`, just to avoid need of deprecations afterwards until we're settled?